Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ruby:3.2.2-alpine3.18@sha256:198e97ccb12cd0297c274d10e504138f412f90bed50c36ebde0a466ab89cf526
+FROM ruby:3.3.0-alpine3.19@sha256:203b3087530e9cb117d8aab9b49bb766253fd8a6606a0d7520a591c7a3d992f7
```

Comment:
This also bumps the version of Alpine, was that on purpose?

Reply:
Yes, it was on purpose. I've updated this PR title & description and will do the same for all the other PR